Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
How about another reply?

---
In reply to: [53844845](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r53844845) [](ancestors = 53844845)